### ISSUE TITLE
big subtitle issue with last alpha 11 build

### ISSUE THREAD

**Issue by janolapin2016** [koying/SPMC#1062]
This last build is very fine but unfortunately it do not show the subtitles on .mkv files...

**Comment by vader913** [comment]
Yeah I have same problem with spmc 17 alpha 11 and alpha 10...with amcodec disabled subtitles work

**Comment by janolapin2016** [comment]
Subtitle work with alpha5 and amcodec enabled but not with alpha10/11 ... 

This is a big bug because unusable for me

**Comment by vader913** [comment]
Cool dude...yeah for me too...amcodec makes playback smooth and without it it's juddery not smooth

**Comment by janolapin2016** [comment]
Not only, I mean original language movies subtitled is the best. 

If we can not do that with an evolved fork its unusable

**Comment by kataliyst** [comment]
Same problem here.  Amcodec needed and subtitles do not work when enabled.  MXIII-G+ SPMC Alpha 10
Had to use Jarvis again...

**Comment by shuugit** [comment]
Can confirm the problem. With amcodec subtitles (embedded or external) do not work. With mediacodec all subtitles work. But some videos have distortions without amcodec...

**Comment by janolapin2016** [comment]
is this issue fixed with last build ?
